src/validation.cpp

```diff
@@ -987,6 +988,7 @@ bool MemPoolAccept::ReplacementChecks(Workspace& ws)
 }
 
 bool MemPoolAccept::PackageMempoolChecks(const std::vector<CTransactionRef>& txns,
+                                         const std::vector<int64_t> &all_tx_vsizes,
```

Comment:
in fef601e2a4d790781fee316c1a55ea48a134a0f1
nit: perhaps pass the vector of `workspaces` instead? It would follow the pattern of the other `MemPoolAccept` functions and be more extendable for future stuff.